### ISSUE TITLE
Cleanup session adapters

### ISSUE THREAD

**Issue by sergeyklay** [phalcon/cphalcon#11341]
It is necessary to clean all Session Adapters from `statsKey` copypasta.
The session adapters don't need this key.

This should be done in <s>2.1.0</s>4.0.0.

**Comment by dzubchik** [comment]
@sergeyklay , maybe milestone should be assigned to this issue?

**Comment by sergeyklay** [comment]
how this will help

**Comment by dzubchik** [comment]
@sergeyklay , sure it will help! Just look at yii2 or symfony issue trackers:

- https://github.com/yiisoft/yii2/issues
- https://github.com/symfony/symfony/issues

It is easy to find out relations between issues and framework component, also it is clear to understand next release issues. So, when I, as developer, want to help I don't understand issues priority, except bug-high, bug-low. IMHO issue tracking is one of the primary things is OS projects and could help improve project continuously. 

Yesterday I have browsed almost all issues in tracker and can say that here we definitely have problems:
* a lot of issues to 1.x version - they can be resolved only by phalcon 1 developers
* issues that have been resolved by pull request, but seemed to be open (from 2013 year) #10239
* issues without participants #3158

Milestones, labels - things that could help to deal with such amount of issues.
